src/components/QuestionCard.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import OpenGraphMeta from "./OpenGraphMeta"
+import { Button } from "react-bootstrap";
+import { IoMdArrowDropup } from "react-icons/io";
+import styled from "styled-components";
+import { useHistory } from "react-router-dom";
+
+const getQuestionById = (question_id) => {
+  /* TODO: Get question data via question ID
+     Expect return result: dict() */
+
+  // retrun {up_votes:"", username:"", ....}
+}
+
+const clickhandle = (history, question_id) => {
+  console.log("enter  ")
+  history.push("/question/" + question_id);
```

Comment:
fix in commit `773de6d`

Reply:
I undo commit 773de6d in be39738. There is nested link in the question card, hence can not use html element.

Reply:
I don't understand, can't you use the React Router `<Link>` component? Seems like it would be clearer and you wouldn't need to pass history around like this.

Reply:
Can't you just do `<Link to={'question/${question_id}'} />`?

Reply:
Read up on nest link to know the reason.

https://stackoverflow.com/questions/9882916/are-you-allowed-to-nest-a-link-inside-of-a-link